src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -69,6 +69,7 @@ public function testCsrfProtectionForFormsEnablesCsrfProtectionAutomatically()
         $this->assertTrue($container->hasDefinition('security.csrf.token_manager'));
     }
 
+    /** @group legacy */
```

Comment:
But you're going to say that hasDefinition won't load the class :) Still, this is a legacy test because it is removed in 3.0. As a rule of thumb, only legacy tests should be removed in 3.0, thus this annotation that "whitelists" the test for removal.